site/docs/v1/tech/apis/_webauthn-response-body-base.adoc

```diff
@@ -0,0 +1,77 @@
+[api]
+ifeval::["{base_field_name}" != "credential"]
+[field]#{base_field_name}# [type]#[Array]#::
+The list of WebAuthn Passkey objects.
+endif::[]
+
+[field]#{base_field_name}.algorithm# [type]#[Long]#::
+The identifier for the signing algorithm used with the passkey. These values are defined by https://www.rfc-editor.org/rfc/rfc9053[RFC 9053]. FusionAuth supports a subset of these algorithms.
```

Comment:
Same comment as above, I thought it was the IANA registry.

Reply:
Should I list the supported algorithms on the response too?

Reply:
I would. Include files are your friend :) 

I'm basing my suggestion off of the `xmlSignatureC14nMethod` which is outlined in both the response and the request here:

https://fusionauth.io/docs/v1/tech/apis/identity-providers/samlv2